src/Pipeline.h

```diff
@@ -158,6 +158,10 @@ class Pipeline {
      * doing. */
     EXPORT void print_loop_nest();
 
+    /** Write out the loop nests specified by the schedule for this
+     * Function for the specified target. */
+    EXPORT void print_loop_nest(Target &target);
```

Comment:
Only for convenience, really. A schedule can necessitate features in the target (by way of device api) and print_loop_nest does a fine job of that by creating a synthetic target with features corresponding to all possible DeviceAPIs. This version of print_loop_nest only provides the user a way to provide a target for the schedule he has specified. It is superfluous really. Will remove if you think it adds confusion about the relationship between schedules and targets.